cdap-runtime-ext-dataproc/pom.xml

```diff
@@ -31,6 +31,11 @@
 
 
   <dependencies>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-common</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
Cannot depend on cdap-common which is more for cdap internal usage. 

See the comment on pom.xml in  https://github.com/cdapio/cdap/pull/13380